src/modules/commander/state_machine_helper.cpp

```diff
@@ -213,11 +213,19 @@ arming_state_transition(struct vehicle_status_s *status,		///< current vehicle s
 						// are measured but are insufficient
 						if (status->condition_power_input_valid && (status->avionics_power_rail_voltage > 0.0f)) {
 							// Check avionics rail voltages
+#ifdef CONFIG_ARCH_BOARD_MINDPX_V2
```

Comment:
1. MindPX hardware simplified power management circuit so it does not distinguish power source. A voltage divider resistor is used in USB1 port to clamp the current, so the original 4.5V threshold will trigger low voltage alarm (when input from USB connection, it's actually about 4.3v). Thus we need to adjust this threshold to avoid the situation.

2. The same reason as in 1), adding an usbconnected flag as the condition will not work in this situation.